lib/tld.js

```diff
@@ -141,7 +141,7 @@ tld.prototype.getRulesForTld = function getRulesForTld (tld, default_rule) {
   }
 
   // Nothing found, apply some default value
-  if (!rules) {
+  if (rules === void 0) {
```

Comment:
`!rules` captures `''`, `null`, `undefined`, `0`, `false` - any falsey value. 

Unfortunately, `undefined` is writable, so not completely safe, whereas `void 0` ensures the `undefined` value is used. See http://stackoverflow.com/questions/4806286/difference-between-void-0-and-undefined. 

Checking against `undefined`/`void` explicitly is the proper way to handle this, rather than a catch-all for a falsey value, then making sure it's not an empty string.